geniza/corpus/templates/corpus/snippets/document_option_label.html

```diff
@@ -0,0 +1,68 @@
+{# template snippet for displaying document label on a form #}
+{# used on document merge form to provide enough information to merge accurately #}
+
+<style>
+    .transcription-container {
+        max-height: 350px;
+        overflow: scroll;
+    }
+
+    ul>li {
+        list-style-type: none;
+    }
+</style>
+
+<div class="merge-document-label">
+    <h2><a target="_blank" href="{% url 'admin:corpus_document_change' document.id %}"
+        title="Go to this document's admin edit page">
+        <img src="/static/admin/img/icon-changelink.svg" alt="Change"></a>
+        {{ document }}</h2>
+    <dl>
+        <dt>Description</dt>
+        <dd>{{ document.description }}</dd>
+        <dt>Languages</dt>
+        <dd>{{ document.all_languages }}</dd>
+        <dt>Tags</dt>
+        <dd>{{ document.all_tags }}</dd>
+        <dt>Notes</dt>
+        <dd>{{ document.notes }}</dd>
+        <dt>Needs Review</dt>
+        <dd>{{ document.needs_review }}</dd>
```

Comment:
I don't think we should display these fields if they are empty. And I think we might want to put the labels and text on the same line so the display is more consolidated (at least for non-note / textarea fields)

Reply:
Done in ffe557a 👍